h2/src/main/org/h2/command/Parser.java

```diff
@@ -3160,8 +3173,7 @@ private Expression readTerm() {
                         String time = currentValue.getString();
                         read();
                         r = ValueExpression.get(ValueTime.parse(time));
-                    } else if (equalsToken("TIMESTAMP", name) ||
-                            equalsToken("TS", name)) {
+                    } else if (equalsToken("TS", name)) {
```

Comment:
you dropped support for TIMESTAMP literals?

Reply:
There is a `TIMESTAMP` block above that reads `TIMESTAMP`, `TIMESTAMP WITH TIME ZONE`, and `TIMESTAMP WITHOUT TIME ZONE`, so this case is not used any more. Before my changes that block was used only for `TIMESTAMP WITH TIME ZONE`.

There are plain `TIMESTAMP '...'` literals in tests and they works properly.

But this code is still required for non-standard `TS`.